sdk/keyvault/keyvault-keys/.eslintignore

```diff
@@ -0,0 +1 @@
+src/core
```

Comment:
Why exclude from eslint?

Reply:
src/core contans a lot of things that break eslint, however: src/core is auto-generated, thus: we have been avoiding changes in src/core. We've been waiting instructions on how to deal with this. I think it's currently in progress. Perhaps @schaabs can provide more information!

Reply:
Please add a comment explaining this